src/manifolds/SymmetricPositiveDefinite.jl

```diff
@@ -1,27 +1,27 @@
-@doc doc"""
+@doc raw"""
     SymmetricPositiveDefinite{N} <: Manifold
 
 The manifold of symmetric positive definite matrices, i.e.
 
 ```math
-\mathcal P(n) =
+𝒫(n) =
 \bigl\{
-x \in \mathbb R^{n\times n} :
-\xi^\mathrm{T}x\xi > 0 \text{ for all } \xi \in \mathbb R^{n}\backslash\{0\}
+x  ∈ ℝ^{n\times n} :
+ξ^\mathrm{T}xξ > 0 \text{ for all } ξ ∈ ℝ^{n}\backslash\{0\}
```

Comment:
ξ is now a symbol for cotangent vectors so using it here may be confusing.

Reply:
Similarly it's used in many places in documentation of vector bundles denoting any vector.